promql/engine.go

```diff
@@ -1500,7 +1500,7 @@ func (ev *evaluator) matrixIterSlice(it *storage.BufferedSeriesIterator, mint, m
 			ev.currentSamples++
 		}
 	}
-	// The seeked sample might also be in the range.
+	// The sought sample might also be in the range.
```

Comment:
This was right.

Reply:
sure? https://www.oxfordlearnersdictionaries.com/definition/english/seek

Reply:
Yes, the method is called Seek and I've never heard of sought time for a hard drive.

Reply:
Me neither, double checked on dictionary! (english is not my native language)
Follow your instructions...